Vision/README.md

```diff
@@ -17,7 +17,7 @@ Ease of gathering personal information spawned business models
 that mined and sold detailed user behaviors, 
 without people’s awareness or consent. 
 The acceleration of global information sharing 
-enabled misinformation to flourish, 
+enabled misinformation & malvertising to flourish, 
```

Comment:
I am uncomfortable adding this term to a document that is intended to be read by a non-native-English-speaking audience, since I'm not sure this is a well-known and natural term yet.  At first, I was just going to suggest "malicious advertising", or at most "malicious advertising ('malvertising')" - but then I realized this doesn't quite fit.  This isn't the only problem, or even the next-worse-problem: misinformation and privacy abuse is probably it.  I think this is why it's taken me a couple of days to respond - I'm not convinced this belongs here, though I agree malicious advertising belongs in our problem definition.  Can we discuss?

Reply:
(also: no ampersands, we should use "and")